assessment.rb

```diff
@@ -1,12 +1,67 @@
 require 'csv'
 # Assessment answers to:
 
+#ran: rails g model Licensee
+#thenran: rake db:migrate
```

Comment:
You'd probably want to populate the migration file with all the columns that you want your table to have, but this is still correct.